ocaml/xapi-cli-server/records.ml

```diff
@@ -3867,7 +3875,10 @@ let sm_record rpc session_id sm =
           ()
       ; make_field ~name:"supported-image-formats"
           ~get:(fun () ->
-            concat_with_comma (x ()).API.sM_supported_image_formats
+            concat_with_comma
```

Comment:
Please don't concatenate with commas, see the comment for the function:

```
(* Some lists are separated with commas and cannot be changed to preserve
   backwards compatibility. Do not use this function on new fields. *)
```

Use concat_with_semi instead

In this case, you might want to use `map_and_concat`